File: 2st/Yeonwoo/외계어 사전.py
def solution(spell, dic):
    answer = 2
    temp = []
    for i in dic:
        if len(i) == len(spell):
            cnt = 0
            for j in spell:
                if j in i:
                    cnt+=1
                if len(spell) == cnt:
                    answer = 1
                    break
                else:
                    answer = 2
    return answer

File: 3st/Juhee/외계어 사전.java
class Solution {
    public int solution(String[] spell, String[] dic) {
        boolean a = false;
        for (String word : dic) {
            int count = 0;
            for (String s : spell) {
                if (word.contains(s)) count++;
            }
            if (count == spell.length) {
                a = true;
                break;
            }
        }
        return a ? 1 : 2;
    }
}
